web/src/message_events.js

```diff
@@ -152,10 +152,16 @@ export function insert_new_messages(messages, sent_by_this_client) {
     }
 
     // sent_by_this_client will be true if ANY of the messages
-    // were sent by this client; notifications.notify_local_mixes
-    // will filter out any not sent by us.
-    if (sent_by_this_client) {
-        compose_notifications.notify_local_mixes(messages, need_user_to_scroll);
+    // were sent by this client except the cases when the current
+    // view is for non-message list views like Inbox;
+    // message_lists.current is undefined for non-message list views like Inbox;
+    // notifications.notify_local_mixes will filter out any not sent by us.
+    if (sent_by_this_client || message_lists.current === undefined) {
```

Comment:
I'm not sure why this isn't still just `sent_by_this_client`?

Reply:
This is no longer needed. The actual bug is fixed in the first commit.

Reply:
Nice, that bug fix is a great find!